erpnext/assets/doctype/location/location_tree.js

```diff
@@ -27,6 +27,31 @@ frappe.treeview_settings["Location"] = {
 			condition: 'frappe.boot.user.can_create.indexOf("Location") !== -1'
 		}
 	],
+	onrender: function (node) {
```

Comment:
Instead of appending it right after the name, we can make it somewhat like how `Account tree` looks like where the area will be set at the right most side of the label?

Take a look at `account_tree.js` file to see how its done there.

Reply:
@Zlash65 New view for location tree: 
![location_tree_view](https://user-images.githubusercontent.com/26640728/47334891-d8039100-d645-11e8-8e7e-0b0ca6c07a01.gif)